html-web-page/index.html

```diff
@@ -72,15 +72,16 @@ <h2>Make all your friends jelly of your stock of jelly!</h2>
   </table>
   <p>Like what you see? Become part of the JOTMC family today!</p>
   <form>
-    <input type="text" name="Your Name">
-    <label id="select">
-      <select id="s" name="reasons">
-        <option value="default">Select an option</option>
+    <label><input placeholder="Your Name" type="text" name="Your Name"></label>
+    <label for="select">
```

Comment:
There is no text labeling these form control elements, so no `label` element is necessary

Reply:
Updated with removed label from 75 and 76
![Update](https://user-images.githubusercontent.com/112663686/195408288-2dec3507-78f6-41d6-84f5-515a562c2116.jpg)